cpp/include/cudf/io/parquet.hpp

```diff
@@ -401,6 +401,9 @@ class parquet_writer_options {
   bool _write_timestamps_as_int96 = false;
   // Column chunks file path to be set in the raw output metadata
   std::string _column_chunks_file_path;
+  /// vector of precision values for decimal writing. Exactly one entry
+  /// per decimal column. Optional unless decimals are being written.
+  std::vector<uint8_t> _decimal_precisions;
```

Comment:
Why is this plural here and singular later? I see both `get_decimal_precision` and `get_decimal_precisions`, both returning vectors.